ultrahdr_api.h

```diff
@@ -66,9 +66,22 @@
  *                                               string. Some bug fixes and introduced one new API
  *                                               which warrants a minor version update. But
  *                                               indicated as a patch update.
+ *   1.1.2           1.1.2                       Some bug fixes, introduced new API and renamed
```

Comment:
I think we can directly go to 1.2.0 since Java APIs are considered as a major change.

Reply:
Done.